### PR TITLE
Adds gunka-cola and Dr. Pecker's DIY Soda

### LINKED ISSUE #20844: How could we make the Robust Softdrinks vending machine more fun/useful?
One thing I love about this game is that almost everything has some super secret, super niche use, and almost all the vendors have some form of junk that you can use effectively or just have fun fucking around with.
The soda vendor seems to be the exception. I just can't find anything to do with it. The only thing it does is give you cans, but Groan's Soda also gives you cans and even cheaper.
I'm not saying REE REMOVE IT, but maybe we can work at least a little fun into it?

Some ideas (looking for more):

 - An expensive large soda bottle that, when emptied, doubles as a large beaker? MetaStation already has this in the form of the soviet soda vendor, which actually sells large beakers for 15$
 - Give more uses to basic soda, even super situational ones? Maybe soda makes gremlins hyperactive? Maybe roaches love a certain type of soda? Maybe it makes gas masks yucky and sticky so the Clown can prank shitters?
 - Contraband/coin sodas that have powergay applications? Maybe a drink that has a bit of hyperzine in it? Maybe it sells bleach?
 - Sodas (but not Groan's) have a 1/10 chance of coming with a SURPRISE COUPON!!! inside? Or a novelty soda that has a glowstick inside?

### PR DESCRIPTION
**Pull request by I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE**
With coder sprites!
dreamseeker_2019-05-11_22-26-15

Gunka-Cola: It costs a whooping 50$, but it comes in a 100u can. Essentially it's an expensive, alternate way to get a large beaker. It's also a way to get black coloring, and the equivalent of 5u hyperzine.

Dr Pecker's DIY Soda: It's a trendy soda you have to make yourself. Comes as its separated components inside a box. Essentially you're paying 95$ for a ghettochem starter kit of 3 vials, 2 small beakers, and 1 large beaker. The soda itself is pretty much useless, it's just Babby's First Taste of Bartendering or something.

Fixes #20844

:cl:
 * rscadd: Added Gunka-Cola Family Sized and Dr. Pecker's DIY Soda to the Robust Softdrinks vending machine.
 * rscadd: Chem-masters now support soda cans and the like.

### CODE REVIEW

**Comment by SonixApache** [comment]
Wait, so this adds small vials of antitox to cola machines?

**Comment by I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE** [comment]
If you want to pay 95$ for 10u of antitox, yeah
Discount Dan's machines can be hacked to get 30u for 10$ tho

File: code/modules/paperwork/paper.dm [hunk 1]
```diff
@@ -539,6 +539,10 @@ var/global/list/paper_folding_results = list ( \
 	name = "paper- 'Suit Dispenser Manual - How to use them?'"
 	info = "Step 1: Place the items that you want the dispenser to dispense on top of one of them, preferably the one bellow this paper.<BR>\nStep 2: Click the dispenser, and choose <b>Define Preset from items on top</b>.<BR>\nStep 3: Click every dispenser you wish to see dispensing, and click <b>Choose a Preset</b>.<BR>\nTo re-use a dispenser, just click <b>Resupply</b>."
 
+/obj/item/weapon/paper/diy_soda
+	name = "paper- 'Instructions'"
+	info = "Thank you for purchasing Dr. Pecker's DIY Soda Kit!<BR>\nIt has been scientifically proven to bring your tastebuds into the delicious state and turn your teeth into a molar solution!<BR>\nNow as you may have guessed, you will have to mix this delicious beverage yourself.<BR>\nDon't worry, it's pretty basic stuff. Just remember to never lick the spoon!<BR>\nFirst, mix the contents of all three <b>small vials</b> into the <b>large flask</b>.<BR>\nThen, mix the contents of the <b>small flasks</b> into the <b>large flask</b>.<BR>\nAnd finally, get ready for our secret trademarked ingredient: <BR>\n<b>The element of surprise!</b>"
```
Comment (I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE) [inline code/modules/paperwork/paper.dm]: please appreciate

Comment (DrSnips) [reply code/modules/paperwork/paper.dm:544]: Post a pic of it in game please.

Comment (I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE) [reply code/modules/paperwork/paper.dm]: dreamseeker_2019-05-12_01-05-39

**Comment by Killette2** [comment]
While this could be a pointless opinion, I think it would be cool if you needed to electrolyze the soda to get your black coloring

**Comment by Kitfox88** [comment]
you can't make me not lick the spoon

**Comment by amikawaii** [comment]
I cant be bothered to find the name of it but this sounds like something that would be better suited for the little gift computer in the front of cargo that sells art supplies and such, not a vending machine.

**Comment by I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE** [comment]
As a child's toy chemistry set? Yeah, possibly. I already coded it as a DIY soda though so you'll have to travel back in time to stop me

**Comment by amikawaii** [comment]
Thats exactly what i thought it was emulating when i read it. I dont know if this is the kind of thing where greyshits are going to be into it every round either.  If you do keep it in a vendor id consider making either of these coin/contraband.

**Comment by aceedex** [comment]
The gift console already has food and drink on it so you could just put it there too

**Pull request merged**